Core/FunctionTree/TreeNode.hpp

```diff
@@ -10,16 +10,16 @@
 #ifndef _TREENODE_HPP_
 #define _TREENODE_HPP_
 
-#include <string>
 #include <complex>
 #include <memory>
+#include <string>
 
-#include "Core/Functions.hpp"
-#include "Core/ParameterList.hpp"
-#include "Core/FitParameter.hpp"
-#include "Core/ParObserver.hpp"
+#include "Functions.hpp"
+#include "ParObserver.hpp"
+#include "ParameterList.hpp"
```

Comment:
In the header file the full path should be specified here

Reply:
Hmm, I prefer includes from the same module to be without the full path. That makes more portable